### PR TITLE
Update homepage of 15 Casks

### PR DESCRIPTION
**Pull request by miccal**
Update homepage of 18 Casks.

Ping @victorpopkov

### CODE REVIEW

File: Casks/toau.rb [hunk 1]
```diff
@@ -2,11 +2,12 @@
   version '1.6'
   sha256 'ef0e3044e1b76677db8f6466f593ed9691c6d457878f1bfa6ea69ff47e3a0228'
 
+  # toauapp.com/download was verified as official when first introduced to the cask
```
Comment (victorpopkov) [inline Casks/toau.rb]: Just `# toauapp.com was verified...`.

File: Casks/thunderbird.rb [hunk 2]
```diff
@@ -60,7 +60,7 @@
   appcast "https://aus5.mozilla.org/update/3/Thunderbird/#{version}/0/Darwin_x86_64-gcc3-u-i386-x86_64/en-US/release/Darwin%2015.3.0/default/default/update.xml?force=1",
           checkpoint: '8a1808dfcba1b3944c707c1e291845f5ae30e71898e0de03333338cafdda2b95'
   name 'Mozilla Thunderbird'
-  homepage 'https://www.mozilla.org/thunderbird/'
+  homepage 'https://www.mozilla.org/en-US/thunderbird/'
```
Comment (victorpopkov) [inline Casks/thunderbird.rb]: Better to leave it as is.

File: Casks/thetube.rb [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
   url "http://download2.equinux.com/files/other/TheTube_#{version}_Web.zip"
   name 'TheTube'
-  homepage 'http://www.equinux.com/'
+  homepage 'http://www.equinux.com/us/start/index.html'
```
Comment (victorpopkov) [inline Casks/thetube.rb]: Since there is a localization part in the URL it's better to leave the link as it was.

File: Casks/texmacs.rb [hunk 4]
```diff
@@ -4,7 +4,7 @@
 
   url "http://www.texmacs.org/Download/ftp/tmftp/macos/TeXmacs-#{version}.dmg"
   name 'GNU TeXmacs'
-  homepage 'http://www.texmacs.org/'
+  homepage 'http://www.texmacs.org/tmweb/home/welcome.en.html'
```
Comment (victorpopkov) [inline Casks/texmacs.rb]: This can be left as is since there is a localization part in the redirect.

**Review by victorpopkov — CHANGES_REQUESTED** [review]

**Comment by miccal** [comment]
Thanks @victorpopkov, I hope it wasn't too annoying for you to review these.

**Review by victorpopkov — APPROVED** [review]

**Comment by victorpopkov** [comment]
> Thanks @victorpopkov, I hope it wasn't too annoying for you to review all of these (there were 220 of them!).

Not really. It had to be done sooner or later.

**Pull request merged**